### PR TITLE
Update repo name in various places

### PR DESCRIPTION
**Pull request by reedy**

### CODE REVIEW

**Comment by compuguy** [comment]
Looks like the manifest also needs to be edited:
`manifest for yp87/leaf2mqtt:6 not found: manifest unknown: manifest unknown`

**Comment by reedy** [comment]
> Looks like the manifest also needs to be edited: `manifest for yp87/leaf2mqtt:6 not found: manifest unknown: manifest unknown`

It looks like it's pretty much a doable find and replace throughout the repo.... Untested though

File: repository.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "name": "LEAF2MQTT Home Assistant add-on repository",
-  "url": "https://github.com/yp87/leaf2mqtt",
-  "maintainer": "YP87 <leaf2mqttaddon@gmail.com>"
+  "url": "https://github.com/kamiKAC/leaf2mqtt",
+  "maintainer": "kamiKAC <kamiKAC@users.noreply.github.com>>"
```
Comment (kamiKAC) [inline repository.json:4]: double '>' at the end
please correct

**Review by kamiKAC — CHANGES_REQUESTED** [review]
Please correct double '>' in repository.json

File: repository.json (hunk 1 above)
Comment (reedy) [reply repository.json]: Done

**Pull request merged**